packages/sol6/src/yang/etsi-nfv-vnf.yang

```diff
@@ -1269,17 +1269,17 @@ submodule etsi-nfv-vnf {
               path "../../nfv:int-virtual-link-desc/nfv:id";
             }
           }
-          container int-cpd {
+          container vdu {
```

Comment:
I am following the naming convention in IFA011 in calling it int-cpd. Do you feel strongly about changing it?

Reply:
No, I dont fell strongly on the naming here, but wanted to give it a similar look as the NSD, where you have a 'container vnf', so I just tried to mirror that:

      choice associated-cpd-id {
          container vnf {
            leaf vnfd-id {
              mandatory true;
              type leafref {
                path "../../../../nfv:vnfd/nfv:id";
              }
              must "boolean(../../../vnfd-id[.=current()])";
            }

            leaf ext-cpd-id {
              mandatory true;
              type leafref {
                path "deref(../vnfd-id)/../nfv:ext-cpd/nfv:id";
              }
            }
          }